### PR TITLE
Implement AttachmentHandler using AliOSS SDK

### PR DESCRIPTION
**Pull request by JohnNiang**
Signed-off-by: johnniang <johnniang@fastmail.com>

### CODE REVIEW

**Review by ruibaby — COMMENTED** [review]
/lgtm

File: src/main/resources/extensions/policy-template-alioss.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    formSchema:
+      - $formkit: text
+        name: bucket
+        label: Bucket
+        validation: required
+      - $formkit: text
+        name: endpoint
+        label: EndPoint
+        validation: required
+      - $formkit: text
```
Comment (ruibaby) [inline src/main/resources/extensions/policy-template-alioss.yaml:25]:
```suggestion
      - $formkit: password
```

File: src/main/resources/extensions/policy-template-alioss.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        name: accessKey
+        label: Access Key
+        validation: required
+      - $formkit: text
```
Comment (ruibaby) [inline src/main/resources/extensions/policy-template-alioss.yaml:29]:
```suggestion
      - $formkit: password
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Comment by JohnNiang** [comment]
/release-note-none
/hold
Until https://github.com/halo-dev/halo/pull/2354 got merged.

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by guqing — COMMENTED** [review]
/approve

**Pull request merged**